test_tools/sigma_verify_rules.py

```diff
@@ -65,6 +102,8 @@ def run_verifier(rules_path, config_file_path):
     return_verified_rules = []
     return_rules_with_problems = []
 
+    ignore = get_sigma_blocklist()
```

Comment:
no path supplied? are you sure this file will then exist in all installations?

Reply:
took care of that.